domain/src/main/java/life/qbic/domain/user/EmailConverter.java

```diff
@@ -0,0 +1,23 @@
+package life.qbic.domain.user;
+
+import javax.persistence.AttributeConverter;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public class EmailConverter implements AttributeConverter<EmailAddress, String> {
+
+  @Override
+  public String convertToDatabaseColumn(EmailAddress emailAddress) {
+    return emailAddress.address();
```

Comment:
```suggestion
    return emailAddress.get();
```
What do you think of introducing a simple get function instead of the address() function?

Reply:
Good idea 👍